src/app.coffee

```diff
@@ -62,6 +63,9 @@ class App
                     deviceName: deviceName
                     password: password
                 Devices.registerDevice options, next
+                filterManager = new FilterManager cozyUrl, deviceName, password
+                filterManager.createOrUpdate {file: true, folder: true}, ->
```

Comment:
You should warn the user if there was an error while creating/updating the view.